services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/health.py

```diff
@@ -0,0 +1,13 @@
+from fastapi import APIRouter, Request
+
+from ..models import ApplicationHealth
+
+health_router = APIRouter()
+
+
+@health_router.get("/health", response_model=ApplicationHealth)
+async def health_endpoint(request: Request) -> ApplicationHealth:
+    return request.app.state.application_health
```

Comment:
Notice that this handler cannot be used for the Dockerfile healthcheck mechanism 
if the service is unhealthy, it should return a client error status !=2XX

In the webserver, this endpoint is named ``/status`` and the root ``/``  or ``/health``returns 200 if  ``request.app.state.application_health.is_healthy`` or status error (e.g. service unavailable) otherwise

The workflow is docker calls ``services/dynamic-sidecar/docker/healthcheck.py`` that queries health entrypoint and the former checks the status code. SEE the code of ``healtcheck.py``

Reply:
I see what you mean, will correct this.